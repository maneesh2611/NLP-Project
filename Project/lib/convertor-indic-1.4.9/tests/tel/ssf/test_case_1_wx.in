1	allari	unk	
2	nareR	unk	
3	mAtlAduwU	unk	
4	,	unk	
5	iMxulo	unk	
6	baMgArrAjuku	unk	
7	sudi	unk	
8	eVkkuvanI	unk	
9	,	unk	
10	awaniwo	unk	
11	paMxeVM	unk	
12	kadiwo	unk	
13	Ulo	unk	
14	eVvarEnA	unk	
15	odipovAlsiMxenanI	unk	
16	annAru	unk	
17	.	unk	
