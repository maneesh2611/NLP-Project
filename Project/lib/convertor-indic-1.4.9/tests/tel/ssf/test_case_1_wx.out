1	అల్లరి	unk	
2	నరేష్	unk	
3	మాట్లాడుతూ	unk	
4	,	unk	
5	ఇందులో	unk	
6	బంగార్రాజుకు	unk	
7	సుడి	unk	
8	ఏVక్కువనీ	unk	
9	,	unk	
10	అతనితో	unk	
11	పందెం	unk	
12	కడితో	unk	
13	ఊలో	unk	
14	ఏVవరైనా	unk	
15	ఓడిపోవాల్సిందేననీ	unk	
16	అన్నారు	unk	
17	।	unk	
